execution/src/windows/execution.cc

```diff
@@ -380,7 +380,7 @@ namespace leatherman { namespace execution {
             if (options[execution_options::throw_on_nonzero_exit]) {
                 throw child_exit_exception("child process returned non-zero exit status.", 127, {}, {});
             }
-            return make_tuple(false, "", "");
+            return make_tuple(false, "", "", 127);
```

Comment:
I pulled it from the exception above, but yes it looks like the right type of error.